extensions/webjars-locator/deployment/src/test/java/io/quarkus/webjar/locator/test/WebJarLocatorDevModeTest.java

```diff
@@ -0,0 +1,58 @@
+package io.quarkus.webjar.locator.test;
+
+import static org.hamcrest.core.Is.is;
+
+import org.jboss.shrinkwrap.api.ShrinkWrap;
+import org.jboss.shrinkwrap.api.asset.StringAsset;
+import org.jboss.shrinkwrap.api.spec.JavaArchive;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.RegisterExtension;
+
+import io.quarkus.test.QuarkusDevModeTest;
+import io.restassured.RestAssured;
+
+public class WebJarLocatorDevModeTest {
```

Comment:
Actually I think this test doesn't actually test dev mode changes (unless I am missing something). The point of the `QuarkusDevModeTest` is to make it easy to change application code and/or configuration and make sure the application works properly with the new changes.

Does that make sense?

Reply:
True, issue is you cannot change dependencies for `QuarkusDevModeTest` (or if there is I couldn't find it), which is basically the only way of modifying the behaviour of this extension (ex: adding the jquery webjar dependency, removing the moment webjar dependency, etc. would be the ideal dev mode test).

Reply:
This test as it currently is basically make sure nothing odd happens when you run in dev mode.

Reply:
I understand that, but we need to make sure that in an application that contains the extension, restart works properly.

So I propose you do the following: add a dummy JAX-RS resource (like other dev mode tests do), and change a string value in it.
Then hit the endpoints again to ensure that everything still works after the restart.